modules/jaggery-apps/portal/gadgets/user_profile/js/gadget.js

```diff
@@ -1,6 +1,6 @@
 function drawPage() {
     var output = "";
-    var u2fScript="<script src=\"u2f-api.js\"></script>";
+    var u2fScript="<script src=\"/portal/gadgets/user_profile/js/u2f-api.js\"></script>";
```

Comment:
The problem in adding url starting with "/" prevent the portal to be re-mapped to different context path via reverse-proxy.
The idea of having the relative URL based on the current page (i.e /portal/gadgets/user_profile/js/")  makes it available even the context path is remapped.
@MohammedEssehemy , can you explain what was to motive to think the original URL is incorrect.

Reply:
To handle wrong script url which cause 404 error in browser console in user_profile page